src/favourites/helpers.ts

```diff
@@ -4,6 +4,8 @@ import { RootState } from '../root/rootReducer';
 export const getPlotSearchFromFavourites = (
   state: RootState
 ): PlotSearch | null => {
+  console.log(state.favourite.favourite);
```

Comment:
I think this little fella has been forgotten here? :thinking:

Linter should warn about these.